### DISCUSSION TITLE
Runtime library

### DISCUSSION THREAD

**Discussion by buongiorgio** [mlocati/gettext-iconv-windows#12] [General]
Hi,
any chance to see in the near future a compiled version of the runtime library for Windows?
Thanks,
Gio

**Comment by mlocati** [comment]
Do you mean a dll library?

**Reply by DanielKlenda-tech** [reply]
What is a dll library, if you do not mind me asking?

**Comment by buongiorgio** [comment]
I'm referring to the packages gettext-runtime and gettext-runtime-dev that are available [here](https://download.gnome.org/binaries/win32/dependencies/). I've developed a C++ application and trying to add internationalization using gettext. The only way I succeeded was using intl.dll and intl.lib found in those packages. The problem is that is an old version and is only available in 32 bit.

**Reply by mlocati** [reply]
So, you need the .lib files, don't you?

**Reply by DanielKlenda-tech** [reply]
I do. ..... I relay do not know. will this help?

**Comment by buongiorgio** [comment]
That's correct.

**Comment by merieinyels** [comment]
Hi, I am in the same situation. I have trying to compile the code my self withouth luck.
Is it planned to add the runtime packages?

**Comment by mlocati** [comment]
I'm very open for a solution

**Reply by merieinyels** [reply]
I am very newby with gettext, it is suppose to be an easy extension or there are tricky issues to take into account?

**Comment by mlocati** [comment]
I've just published a new release: you can download the .h / .lib files (for Microsoft Visual C) and the .h / .a files (for gcc).

See https://mlocati.github.io/articles/gettext-iconv-windows.html
